src/components/Checkout/DonationItem.vue

```diff
@@ -222,23 +224,14 @@ export default {
 		formattedAmount() {
 			return numeral(this.amount).format('$0,0.00');
 		},
-		donationTitle() {
-			return this.checkoutDonationLendingCostExperiment === 'variant-b'
-				? 'Kiva love'
-				: 'Donation to Kiva';
-		},
 		donationTagLine() {
 			/* eslint-disable max-len */
-			return this.checkoutDonationLendingCostExperiment === 'variant-b'
-				? `An optional 15% donation covers Kiva's costs for ${this.loanCount > 1 ? 'these loans' : 'this loan'}`
-				: `${this.loanCount > 1 ? 'These loans cost' : 'This loan costs'} Kiva more than ${numeral(Math.floor(this.loanReservationTotal * 0.15)).format('$0,0')} to facilitate. Will you help us cover our costs?`;
+			const coverOurCosts = `${this.loanCount > 1 ? 'These loans cost' : 'This loan costs'} Kiva more than ${numeral(Math.floor(this.loanReservationTotal * 0.15)).format('$0,0')} to facilitate. Will you help us cover our costs?`;
+			return this.checkoutDonation100TextExperiment === 'variant-b'
+				? `${coverOurCosts} 100% of every dollar lent goes to funding loans.`
+				: coverOurCosts;
 			/* eslint-enable max-len */
 		},
-		donationDetailsLink() {
-			return this.checkoutDonationLendingCostExperiment === 'variant-b'
-				? 'How Kiva uses donations'
-				: 'Learn more';
-		},
```

Comment:
I moved these to `data` for now, I'm sure we'll experiment on this section again and it'd be good to keep the variables

Reply:
all good